components/resources/library/src/androidMain/kotlin/org/jetbrains/compose/resources/AndroidContextProvider.kt

```diff
@@ -37,11 +38,34 @@ fun PreviewContextConfigurationEffect() {
     }
 }
 
+/**
+ * Sets the Android instrumented context.
+ *
+ * @param context The Android context to be set as the instrumented context.
+ *                This context will be used by a ResourceReader to read test assets.
+ *
+ * Example usage:
+ * ```
+ * fun configureTestEnvironment() {
+ *     setAndroidInstrumentedContext{
+ *         InstrumentationRegistry.getInstrumentation().context
+ *     }
+ * }
+ * ```
+ */
+@ExperimentalResourceApi
```

Comment:
Let's create an issue for removing Experimental with target = 1.8

Reply:
https://youtrack.jetbrains.com/issue/CMP-6547